mitiq/pec/test/test_pec_sampling.py

```diff
@@ -0,0 +1,115 @@
+# Copyright (C) 2020 Unitary Fund
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Tests related to mitiq.pec.sampling functions."""
+
+from pytest import mark
+import numpy as np
+
+from cirq import (
+    Circuit,
+    Operation,
+    Gate,
+    LineQubit,
+    X,
+    Y,
+    Z,
+    CNOT,
+    depolarize,
+)
+
+from mitiq.pec.utils import _simple_pauli_deco_dict, DecoType
+from mitiq.pec.sampling import sample_sequence, sample_circuit
+from mitiq.utils import _operation_to_choi, _circuit_to_choi
+
+BASE_NOISE = 0.02
+DECO_DICT = _simple_pauli_deco_dict(BASE_NOISE)
+DECO_DICT_SIMP = _simple_pauli_deco_dict(BASE_NOISE, simplify_paulis=True)
+NOISELESS_DECO_DICT = _simple_pauli_deco_dict(0)
+
+# Simple 2-qubit circuit
+qreg = LineQubit.range(2)
+twoq_circ = Circuit(
+    X.on(qreg[0]),
+    CNOT.on(*qreg),
+)
+
+
+@mark.parametrize("gate", [X, Y, Z, CNOT])
+def test_sample_sequence_types(gate: Gate):
+    num_qubits = gate.num_qubits()
+    qreg = LineQubit.range(num_qubits)
+    for _ in range(1000):
+        imp_seq, sign, norm = sample_sequence(gate.on(*qreg), DECO_DICT)
+        assert all([isinstance(op, Operation) for op in imp_seq])
+        assert sign in {1.0, -1.0}
+        assert norm > 1
+
+
+def test_sample_circuit_types(circuit: Circuit = twoq_circ):
```

Comment:
Similar comment that argument can be removed and used in body of function if not being parameterized. (Unless there's a reason to do so.)